docs/runtime/agents/tech-writer.md

```
@@ -3,22 +3,15 @@ name: tech-writer
 description: Technical Writer. Use for user documentation, operator manuals, API/function-block references, how-to guides, changelogs, and release notes. Prose artifacts intended for human readers outside the agent team.
 model: sonnet
 canonical_source: .claude/agents/tech-writer.md
-canonical_sha: c90e620af4b82d0501acc21fa96b8655d7d12d68
+canonical_sha: fc426200f9766813c15276ed3e39dc8778616a46
 generator: scripts/compile-runtime-agents.sh
 generator_version: 0.2.0
 classification: generated
 ---
 
 ## Project-specific local supplement
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)